tests/test_wave_01.py

```diff
@@ -130,14 +131,14 @@ def test_update_task_not_found(client):
 
     # Assert
     assert response.status_code == 404
-
-    raise Exception("Complete test with assertion about response body")
+    assert response_body == {"details": f"Could not find task with id #1"}
```

Comment:
Minor note, the value in the dictionary here does not need to be an f-string, because there are no variables in it.  This line is the same as:
```suggestion
    assert response_body == {"details": "Could not find task with id #1"}
```